gradle/root/fatJars.gradle

```diff
@@ -52,6 +52,8 @@ dependencies {
   netcdfAll project(':cdm:cdm-image')
   netcdfAll project(':cdm:cdm-radial')
   netcdfAll project(':cdm:cdm-misc')
+  netcdfAll project(':cdm:cdm-s3')
+  netcdfAll project(':cdm:cdm-zarr')
   netcdfAll project(':bufr')
```

Comment:
we're not ready to include these in the fatJar by default :)

Reply:
@haileyajohnson, I know. Half the time when I'm pushing my copy of the repo from Xcode to Github, I forget to uncheck the box for that file.

Reply:
I figured :)